flow/designs/gf12/bp_quad/config_synth.mk

```diff
@@ -0,0 +1,73 @@
+export DESIGN_NICKNAME = bp_quad
```

Comment:
What is the difference between config_hier and config_synth?

Reply:
config_synth was used to generate the cached netlist. I pushed it into the repo so if necessary we can experiment with ABC options and hierarchy flattening settings